fitwsarah_frontend/src/views/AdminPanelPage/AdminAppointments.js

```diff
@@ -158,33 +165,60 @@ function AdminAccounts() {
                 "Content-Type": "application/json"
             })
         })
-        .then((response) => {
-            if (!response.ok) {
-                throw new Error('Network response was not ok ' + response.statusText);
-            }
-            return response.json();
-        })
-        .then((data) => {
-            console.log(data);
-            setAppointments(data);
-        })
-        .catch((error) => {
-            console.log(error);
-        });
+            .then((response) => {
+                if (!response.ok) {
+                    throw new Error('Network response was not ok ' + response.statusText);
+                }
+                return response.json();
+            })
+            .then((data) => {
+                console.log(data);
+                setAppointments(data);
+            })
+            .catch((error) => {
+                console.log(error);
+            });
     };
 
     const updateAppointmentStatus = (appointmentId, status) => {
-        fetch(`${process.env.REACT_APP_BASE_URL}/api/v1/appointments/${appointmentId}/${status}`, {
+        fetch(`${process.env.REACT_APP_BASE_URL}/api/v1/appointments/${appointmentId}/cancelled`, {
             method: "PUT",
             headers: new Headers({
                 Authorization: "Bearer " + accessToken,
                 "Content-Type": "application/json"
             }),
             body: JSON.stringify(status)
+
         })
             .then((response) => {
                 if (!response.ok) {
-                    throw new Error('Network response was not ok ' + response.statusText);
+                    throw new Error('Network response was not ok');
+                }
+                return response.json();
+            })
+            .then((data) => {
+                console.log(data);
+                getAppointments();
+            })
+            .catch((error) => {
+                console.log(error);
+
+            });
+    }
+
+
+    const handleAppointmentRequest = (appointmentId, status) => {
+        fetch(`${process.env.REACT_APP_BASE_URL}/api/v1/appointments/${appointmentId}/scheduled`, {
+            method: "PUT",
+            headers: new Headers({
+                Authorization: "Bearer " + accessToken,
+                "Content-Type": "application/json"
+            }),
+            body: JSON.stringify(status)
```

Comment:
Could I ask why are you only passing the status in the json request?

Reply:
This method only changes the status of the appointment that's why I'm passing it.